### PR TITLE
example projects: link to test examples for go and python

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @nicks, @landism,

Please review the following commits I made in branch maiamcc/tests-for-examples:

677c829f3c808dcc192712f741b16656bbc09844 (2021-03-26 16:24:27 -0400)
example projects: link to test examples for go and python

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by maiamcc** [comment]
Relies on https://github.com/tilt-dev/tilt-example-nodejs/pull/12 and https://github.com/tilt-dev/tilt-example-go/pull/15

File: docs/example_go.md [hunk 1]
```diff
@@ -294,6 +294,11 @@ services in a Tiltfile and exits successfully if they're healthy.
 - [CircleCI config](https://github.com/tilt-dev/tilt-example-go/blob/master/.circleci/config.yml)
 - [Test script](https://github.com/tilt-dev/tilt-example-go/blob/master/test/test.sh)
 
+### Tests in Tilt
+You can run your tests, formatters, linters, and other tools directly from Tilt, so that you don't have to tab between a million windows and remember to run a bunch of different commands. For more information, see the [Tests in Tilt docs](https://docs.tilt.dev/tests_in_tilt.html). We also provide [an extension](https://github.com/tilt-dev/tilt-extensions/tree/master/tests/golang) to streamline setup of the most common Go test configurations. Here's an example of how you might set up your Go tests in Tilt:
```
Comment (nicks) [inline docs/example_go.md:298]: i don't think the tests in tilt doc are officially public yet (i.e., in the sidebar?)

**Comment by maiamcc** [comment]
> i don't think the tests in tilt doc are officially public yet (i.e., in the sidebar?)

as per slack discussions, we're waiting to merge this PR until the Tests feature is public (i.e. in the sidebar + api reference) or til there is more of a release plan. @ellenkorbes will coordinate when these actually go out!

**Comment by nicks** [comment]
closing this as stale, we may re-open it someday!

**Pull request closed without merging**